### DISCUSSION TITLE
Is 4.x good enough for general use, such as Linux distro packages?

### DISCUSSION THREAD

**Discussion by yan12125** [buildbot/buildbot#8075] [Q&A]
4.x has been out for some time, while it seems there are still several unresolved regressions [1]. Is it good enough for general use? For example, I wonder if it's good to push 4.x to Arch Linux stable repos or not. Currently I keep it in testing repos [2].

[1] https://github.com/buildbot/buildbot/issues?q=is%3Aissue+is%3Aopen+label%3Abug
[2] https://archlinux.org/packages/extra-testing/any/buildbot/

**Comment by yan12125** [comment]
@p12tic How do you think?

**Reply by p12tic** [reply]
@yan12125 Sorry for a late response. I think let's wait for a bit as there are new bugs reports about things that I thought were fixed already.

**Reply by yan12125** [reply]
Got it. I will drop 4.x from Arch testing repos and keep 3.x only, until 4.x becomes good enough.

**Reply by yan12125** [reply]
Thinking again, dropping 4.x from Arch testing repos may not be a good idea. Users are likely to going to downgrade buildbot, as pacman (Arch's package manager) will complain about something like local versions are newer than those in repos. However, downgrading is not supported https://github.com/buildbot/buildbot/issues/7420
